flac.md

```diff
@@ -987,6 +989,58 @@ Change controller: Internet Engineering Task Force
 Provisional registration? (standards tree only): NO
 ```
 
+## Application ID Registry
+
+This document creates a new IANA registry called the "FLAC Application Metadata Block ID" registry. The values correspond to the 32-bit identifier described in (#application).
+
+To register a new Application ID in this registry, one needs an Application ID, a description, optionally a reference to a document describing the Application ID and a Change Controller (IETF or email of registrant). The Application IDs are to be allocated according to the "First Come First Served" policy [RFC8126]. An Application ID can be any 32-bit value, but is often composed of 4 ASCII characters, to be human-readable.
+
+The FLAC Application Metadata Block ID registry is assigned the following initial values.
+
+Application ID   | ASCII rendition (if available) | Description                        | Specification       | Change controller
+:----------------|:-------------------------------|:-----------------------------------|:--------------------|:---------
+0x41544348       | ATCH                           | FlacFile                           | [@FlacFile]         | IETF
```

Comment:
I'm getting "no page found" for flacFile. Is that still working for you?

Reply:
Works fine on my end. Did you take the whole url or just the last bit? This should work: https://web.archive.org/web/20071023070305/http://firestuff.org:80/flacfile/